ofl/natasans/article/ARTICLE_en_us.html

```diff
@@ -0,0 +1,13 @@
+<p><strong>Nata Sans</strong> is a grotesque sans-serif typeface with a subtle humanist tone, designed by Daniel Uzquiano Cruz.</p>
+
+<p>It was developed with user interfaces in mind, featuring a generous x-height, wide glyphs, and short ascenders and descenders that promote a relaxed and readable rhythm. Nata Sans presents a restrained yet distinct personality, making it a viable alternative to Helvetica in both digital and print contexts.</p>
+
+<p>The typeface includes nine predefined weights from Thin to Black, and is available as a variable font with a weight axis ranging from 100 to 900. All weights interpolate smoothly thanks to three master drawings and cover 1022 glyphs.</p>
+
+<p>Nata Sans supports 219 Latin-based languages spoken across 212 countries. It meets 100% of the <strong>Latin Plus</strong> character set, making it fully equipped for a wide range of global applications.</p>
```

Comment:
There are several "Latin Plus" glyph set definitoins floating around, I think its better to say "the Google Fonts Latin Plus glyph set"